### ISSUE TITLE
Support multiple intents per message

### ISSUE THREAD

**Issue by nabihach** [RasaHQ/rasa_nlu#374]
Does Rasa allow multiple intents per example? For instance, the sentence
"Please go to the office and pick up the files" has two intents: go_to_office and pick_up_files. How can I capture this in Rasa?

**Comment by amn41** [comment]
It's something we would like to support in the future, but don't have currently I'm afraid. If you would like to start work on contributing this that would be awesome! You'd get plenty of support from the maintainers :)

**Comment by znat** [comment]
What about this very bad and hacky idea?
Since you get multiple intents with confidence, you could add the sentence twice, each time with a different intent to the training set. Confidence indices should be close enough.

**Comment by srijiths** [comment]
Another approach would be split sentences in to possible clauses and then find intent for each clause. Then combine the results.  In above example "Please go to the office" and "pick up the files" will be the two clauses. Easy hack :) . I am planning to open source a project on clause identification for English.

**Comment by tmbo** [comment]
@srijiths awesome, please keep us updated on how that goes.

**Comment by tmbo** [comment]
@srijiths how's that project coming along? ;)

**Comment by srijiths** [comment]
@tmbo you can close this issue. I have started it. Will let you know.

**Comment by wrathagom** [comment]
Re-read, sounds like the Rasa team wants to support it, so leave it open as a place holder either for their work or yours. I'll add the help wanted label.

**Comment by znat** [comment]
Constituency parsers don't seem to be widely available. Neither spacy, syntaxnet, nor commercial products such MS cognitive services or event Googles NLP api offers this.

I am trying to hack it as follows and that seems to work even if I a not sure how it scales

```
## intent: pickupfiles
- Pick up the files

## intent: go_to_office
- Go to the office

## intent: go_to_office_pickupfiles
- Go to the office and pick up the files

```

**Comment by tmbo** [comment]
This is now supported through the tensorflow embedding pipeline. The syntax is `go_to_office+pickupfiles` though.
